### PR TITLE
Botolytics

### PR DESCRIPTION
**Pull request by LindaWeimann**
# Summary

Adding Bot'O'lytics Quickstart with 1 Dashboard.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [didn't work, sanitized manually] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard? 

### Alerts

- [no alerts in the quickstart] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/bot'o'lytics/config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+id:
```
Comment (brammerl) [inline quickstarts/bot'o'lytics/config.yml:1]: please remove this

File: quickstarts/bot'o'lytics/config.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - bot
+  - traffic analysis
+  - bots
+  - fraud detection
+
+documentation:
+  - name: Installation docs
+    url: docs.newrelic.com/introduction-apm
+    description: Add APM data to use this Quickstart
+
```
Comment (brammerl) [inline quickstarts/bot'o'lytics/config.yml:31]: In order to install the dashboards there needs to be a data source created and assigned to this quickstart. 

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources

File: quickstarts/bot'o'lytics/config.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+id:
+slug: bot-o-lytics
+title: Bot'O'lytics
+description: |
+  ## Bot'O'lytics
+
+  On every well used website there are not only real users, but also many many bots. Those can be legitimate bots, like search bots, synthetic checks or other automated calls to your service. But even legitimate bots can distort your service's vitals in a way that they don't anylonger represent your real customer's experience.
```
Comment (seemantk) [inline quickstarts/bot'o'lytics/config.yml:7]:
```suggestion
  On every well used website there are not only real users, but also many many bots. Those can be legitimate bots, like search bots, synthetic checks or other automated calls to your service. But even legitimate bots can distort your service's vitals in a way that they no longer represent your real customer's experience.
```

File: quickstarts/bot'o'lytics/config.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  Being able to distinguish real traffic from bot traffic is cruitial in various cases, for example when examining a marketing campain or when analysing your services vitals or service levels.
```
Comment (seemantk) [inline quickstarts/bot'o'lytics/config.yml:8]:
```suggestion
  Being able to distinguish real traffic from bot traffic is crucial in various cases, for example when examining a marketing campaign or when analyzing your services vitals or service levels.
```

File: quickstarts/bot'o'lytics/config.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+  ### How to use Bot'o'lytics
+
+  This Quickstart works on top of your APM data. We have included the most common bots, but you can and should customize the dashboard with bots specific to your service.
+
+summary: |
+  On every well used website there are not only real users, but also many many bots. Those can be legitimate bots, like search bots, synthetic checks or other automated calls to your service. But even legitimate bots can distort your service's vitals in a way that they don't anylonger represent your real customer's experience.
```
Comment (seemantk) [inline quickstarts/bot'o'lytics/config.yml:15]:
```suggestion
  On every well used website there are not only real users, but also many many bots. Those can be legitimate bots, like search bots, synthetic checks or other automated calls to your service. But even legitimate bots can distort your service's vitals in a way that they no longer represent your real customer's experience.
```

File: quickstarts/bot'o'lytics/config.yml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  Being able to distinguish real traffic from bot traffic is cruitial in various cases, for example when examining a marketing campain or when analysing your services vitals or service levels. The Bot'o'lytic Dashboard enables you to analyse your traffic for bot traffic.
```
Comment (seemantk) [inline quickstarts/bot'o'lytics/config.yml:16]:
```suggestion
  Being able to distinguish real traffic from bot traffic is crucial in various cases, for example when examining a marketing campaign or when analyaing your services vitals or service levels. The Bot'o'lytic Dashboard enables you to analyze your traffic for bot traffic.
```

File: quickstarts/bot'o'lytics/config.yml [hunk 7]
```diff
@@ -0,0 +1,33 @@
+id:
+slug: bot-o-lytics
+title: Bot'O'lytics
+description: |
+  ## Bot'O'lytics
```
Comment (seemantk) [inline quickstarts/bot'o'lytics/config.yml:5]:
```suggestion
  ## Bot'o'lytics
```

File: quickstarts/bot'o'lytics/config.yml [hunk 8]
```diff
@@ -0,0 +1,33 @@
+id:
+slug: bot-o-lytics
+title: Bot'O'lytics
```
Comment (seemantk) [inline quickstarts/bot'o'lytics/config.yml:3]:
```suggestion
title: Bot'o'lytics
```

**Review by seemantk — CHANGES_REQUESTED** [review]
hi Heiko!  I'm so glad to see this quickstart!!  Just some language changes and we should be good to go on this

File: data-sources/bot'o'lytics/config.yml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+id: bot'o'lytics
```
Comment (josephgregoryii) [inline data-sources/bot'o'lytics/config.yml:1]: Can we update this to not have any `'`?

File: quickstarts/bot'o'lytics/config.yml [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+  - bots
+  - fraud detection
+
+documentation:
+  - name: Installation docs
+    url: docs.newrelic.com/introduction-apm
+    description: Add APM data to use this Quickstart
+
+dashboards:
+  - bot'o'lytics
```
Comment (josephgregoryii) [inline quickstarts/bot'o'lytics/config.yml:34]: We would want to update this as well!

File: quickstarts/bot'o'lytics/config.yml [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+dataSourceIds:
+  - bot'o'lytics
```
Comment (josephgregoryii) [inline quickstarts/bot'o'lytics/config.yml:37]: here as well

**Review by josephgregoryii — CHANGES_REQUESTED** [review]
Overall the PR looks good. One thing I would like to change is a couple of naming updates with the files and directories:

- Having a `'` in the directory names may lead to some weird breakage in our code with escaped characters. One idea i may suggest is to replace any `'` with dashes (`-`), or remove them completely.
  - Same with file names, `bot'o'lytics` -> `botolytics` or `bot-o-lytics`
- Can we update the screenshot names to describe the image more?
  - e.g., `Botolytics-bots-vs-real.png`, or something along those lines!

**Review by zstix — CHANGES_REQUESTED** [review]
Looks like this PR has a few things that still need to be updated:

- Remove the `'` from _all_ filename locations (folder names still include it)
- You have two quickstarts with the same name or ID, remove one
- The quickstart(s) are missing an icon. This is a required item, please add one
- You have two _dashboards_ with the same name or ID, please remove one or update the names

My guess is that you created a new version of the quickstart and dashboard without the `'` character, but didn't remove the old files. Removing those should help with most of the issues (leaving just the icon to resolve).

**Comment by brammerl** [comment]
Leaving open for now, @seemantk will converse with Heiko about re-creating an updated quickstart

**Comment by pkosanam** [comment]
Closing this PR, as it got updated on PR- https://github.com/newrelic/newrelic-quickstarts/pull/1932

**Pull request closed without merging**